src/content/docs/en/tutorial/1-setup/index.mdx

```diff
@@ -66,6 +66,6 @@ Every time you commit a change to GitHub, a notification will be sent to Netlify
 ## Checklist
 
 <Checklist>
-- [ ] Let's get set up for building an Astro project!
+- [ ] I'm ready to prepare a deployment environment for an Astro project!
```

Comment:
```suggestion
- [ ] I'm ready to prepare a development environment for an Astro project!
```
Since it's not strictly deployment, and "development and deployment" feels a little long, I think I'll swap this for development.